lib/bluetooth/encrypted_ad_data/encrypted_ad_data.h

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/toolchain/gcc.h"
+#include <stddef.h>
+#include <stdint.h>
+
+#define EAD_AAD_SIZE	    1  /**< Additional Authenticated Data size in bytes */
+#define EAD_RANDOMIZER_SIZE 5  /**< Randomizer size in bytes */
+#define EAD_NONCE_SIZE	    13 /**< nonce size in bytes */
+#define EAD_KEY_SIZE	    16 /**< Key size in bytes */
+#define EAD_IV_SIZE	    8  /**< Initialisation Vector size in bytes */
+#define EAD_MIC_SIZE	    4  /**< MIC size in bytes */
+
```

Comment:
If this is a public API it shouldn't be coming up with a new namespace like that (at least not without formally requesting/allocating one). The appropriate namespace should be `bt_*` like for all other Bluetooth APIs, and the header file should go in the same place as other public Bluetooth APIs.